keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ApplicationLoadBalancerHandler.kt

```diff
@@ -69,7 +72,7 @@ class ApplicationLoadBalancerHandler(
               spec.moniker.app,
               description,
               listOf(spec.toUpsertJob()),
-              OrchestrationTrigger(resource.id.toString())
+              OrchestrationTrigger(correlationId = resource.id.toString(), notifications = notifications)
```

Comment:
I wonder if we can do this in a centralized place. I worry we'll forget to do notifications for a particular resource type or something stupid. (Future change, just thinking aloud).

Reply:
Hmm good point. I could just make it required to pass in, that way you have to think about it? that would probably solve the issue enough to help ward off future mistakes.

Reply:
Oo nice call. Found a place where I missed one (security group create).

Reply:
Let's think about it in a future change.